### PR TITLE
Assignment-5 Uploading

### PR DESCRIPTION
**Pull request by VIP5614**

### CODE REVIEW

File: Vish/assignment-5/VishnuA5_1.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                    <li>I have done my 12th in DAV Inter college and 10th in our DAV Inter College.</li>
+                    <li>Talking about my family, my family consists of 7 members including me and my elder brother. </li>
+                    <li>My father is a farmer and my mother is a housewife.</li>
+                    <li>My hobbies are cooking, listening to music and playing the shuttle.</li>
+                </ol>
+            </div>
+            <h3 class="left-content mau">MoreAbout Me</h3>
+            <div class="about MoreAbout">
+                <ul type=disk>
+                    <li><a href="https://www.facebook.com/profile.php?id=100008154193219" target="_blank"> My Resume </a></li>
```
Comment (sanketmod) [inline Vish/assignment-5/VishnuA5_1.html]: change the shape of  ul element from bullets to square  colour them and apply border.

**Review by sanketmod — CHANGES_REQUESTED** [review]

**Review by pratikshaghule — APPROVED** [review]

**Review by mayank30041995 — APPROVED** [review]

File: Vish/assignment-5/VishnuA5_1.html (hunk 1 above)
Comment (VIP5614) [reply Vish/assignment-5/VishnuA5_1.html]: kya sanket main samjha nahi

Comment (sanketmod) [reply Vish/assignment-5/VishnuA5_1.html]: thume ul ke bullets ko square shape dena hai aur usko color aur border dena hai

Comment (VIP5614) [reply Vish/assignment-5/VishnuA5_1.html]: i change it

Comment (VIP5614) [reply Vish/assignment-5/VishnuA5_1.html]: maine CSS se add kar diya hai use

**Review by poonam-khode — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**